### ISSUE TITLE
can i disabel verbosity ?

### ISSUE THREAD

**Issue by almadrasa** [Genymobile/gnirehtet#416]
![image](https://user-images.githubusercontent.com/34221874/138338083-7ea59bbb-8604-4560-9f76-a3debda020f0.png)
it log all redirected connections , will that effect the internet latency ??
and can i disable it ?

**Comment by VaradGupta23** [comment]
Yes, you can disable or reduce the verbosity of the logs, but it requires modifying the Gnirehtet source code or rebuilding it with a different logging level.

🔧 How to reduce log verbosity:
The log messages (like TcpConnection and UdpConnection) are generated by Java logging (Logback or java.util.logging) in the desktop gnirehtet executable.

Option 1: Edit logging configuration (preferred)
In the source code, look for the logging config file or the logger initialization (like Logger.getLogger(...)), and set the level to WARNING or SEVERE instead of INFO.

If you're using the Rust version, look for code like:

env_logger::Builder::new().filter_level(LevelFilter::Info).init();
And change it to:

env_logger::Builder::new().filter_level(LevelFilter::Warn).init();
Option 2: Suppress logs using command line (if possible)
If your OS or terminal supports it, you can filter logs like:


./gnirehtet run 2>&1 | grep -v "TcpConnection"
 Will it affect latency?
 No, the logging itself won’t affect latency in any noticeable way, but if:

You are logging to disk intensively

Or running on a very low-spec machine

...then disk I/O could cause micro-lags. But this is rare and not typically a problem unless you're logging thousands of lines per second.
